src/main/scala/cromwell/logging/CromwellLogger.scala

```diff
@@ -17,7 +17,7 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
 
     val highlightedMessage = event.getFormattedMessage
       .replaceAll("UUID\\((.*?)\\)", TerminalUtil.highlight(2, "$1"))
-      .replaceAll("`(.*?)`", TerminalUtil.highlight(9, "$1"))
+      .replaceAll("`(.*?)`", TerminalUtil.highlight(99, "$1"))
```

Comment:
The number is fairly arbitrary anyway because terminals can set their own themes so there's very little guarantee about how colors will show up.  I think most implementations honor the first 8 pretty closely, though.  That's where most programs get the yellow for warnings and the red for errors.  Maybe we can choose one of the first 8 color codes, which looks like this on my machine:

<img width="481" alt="screen shot 2015-08-03 at 6 56 57 am" src="https://cloud.githubusercontent.com/assets/58551/9036178/f360062e-39ac-11e5-9aed-72d4f81d2626.png">